rewrite-java-core/src/main/resources/META-INF/rewrite/rewrite.yml

```diff
@@ -3,7 +3,8 @@ name: com.azure.rewrite.java.core.MigrateAzureCoreSamplesToAzureCoreV2
 displayName: Migrate azure-core samples to azure-core-v2
 description: This recipe migrates the samples in azure-core to azure-core-v2
 recipeList:
-  - org.openrewrite.java.ChangePackage:
-      oldPackageName: com.azure
-      newPackageName: io.clientcore
-      recursive: true
\ No newline at end of file
+  - org.openrewrite.text.FindAndReplace:
```

Comment:
The recipe looks good. 
We are missing tests as mentioned in the original PR comments 
https://github.com/Azure/azsdk-java-rewrite-recipes/pull/6#discussion_r1698988564
https://github.com/Azure/azsdk-java-rewrite-recipes/pull/6#discussion_r1700637066